.github/workflows/container.yaml

```
@@ -0,0 +1,85 @@
+name: Container Image
+
+on:
+  # Event triggered upon completion of the `release.yaml` workflow
+  workflow_run:
+    workflows: ['Release']
+    types:
+      - completed
+
+  # Manual trigger
```

Likewise, while at it this might be obvious in some parts or helpful in others, (_the maintainer can trim the comment once they review, but it should otherwise make it easy to quickly grok during review_):

```suggestion
  # Manual trigger via GH Actions page, runs this workflow from a selected tag or branch,
  # which also affects the git clone for the `Dockerfile` used, and `github.ref_type` (branch / tag)
```